fvp.mk

```diff
@@ -173,8 +234,15 @@ grub-clean:
 ################################################################################
 # Boot Image
 ################################################################################
+
+# Force to build linux before attempting to build the image as we need the
+# ftpm module for the latter.
+.PHONY: final-img
+final-img: linux
```

Comment:
Would it be possible to do this with real dependencies instead of using recursive make?

Reply:
Fixed